docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-monitoring-stack.adoc

```diff
@@ -65,18 +62,12 @@ endif::[]
 . Remove the undercloud container images:
 +
 ----
-# remove the base image
-cephadm shell -- ceph config rm mgr mgr/cephadm/container_image_base
-# remove the undercloud images associated to the monitoring
-# stack components
+$ cephadm shell -- ceph config rm mgr mgr/cephadm/container_image_base
 for i in prometheus grafana alertmanager node_exporter; do
     cephadm shell -- ceph config rm mgr mgr/cephadm/container_image_$i
 done
 ----
-
++
 [NOTE]
-In the example above, in addition to the monitoring stack related
-container images, we update the config entry related to the
-container_image_base. This has an impact on all the {Ceph} daemons that rely on
-the undercloud images.
-New daemons will be deployed using the new/default {Ceph} image.
+In addition to updating the container images related to the monitoring stack, you must update the configuration entry related to the `container_image_base`. This has an impact on all the {Ceph} daemons that rely on the undercloud images.
```

Comment:
Is this something we want to move in a dedicated prereq section?

Reply:
@fmount Do you mean that this note should go in the yet-to-be-created Ceph prerequisites section in the Planning chapter? Or do you mean it should not be a note but a step in this prerequisites section instead?

Reply:
Yes, it makes sense to do that step as a prereq instead of reaching here and finding a problem with container images that should be switched in the cluster. Eventually, we can have a note pointing to that section.